src/components/CityCreateForm/StyledCityCreate.js

```diff
@@ -92,28 +81,25 @@ export const StyledTotalPrice = styled.div`
 
 export const StyledSection = styled.section`
   display: flex;
+  flex-direction: column;
   align-items: flex-start;
-  gap: 0.5rem;
-  padding: 0.5rem;
+  flex-wrap: wrap;
```

Comment:
Not a request for change, but you can also combine
`flex-direction` & `flex-wrap` together by using `flex-flow`:

```suggestion
  flex-flow: column wrap;
```